app/models/security/synth_combobox_option.rb

```diff
@@ -0,0 +1,22 @@
+class Security::SynthComboboxOption
+  include ActiveModel::Model
+
+  attr_accessor :symbol, :name, :logo_url, :exchange_acronym
+
+  class << self
+    def find_in_synth(query)
+      Provider::Synth.new(ENV["SYNTH_API_KEY"])
```

Comment:
I would `include Providable` in this class and access Synth as `security_prices_provider` here so this will work for self hosters who have Synth setup:

```
security_prices_provider.search_securities( ... )
```